### PR TITLE
locale: freeze strings and update active translation files

### PR DESCRIPTION
**Pull request by patricklodder**
- Updates `bitcoinstrings.cpp` and `bitcoin_en.ts` base files
- Removes all translations that [we stopped maintaining with 1.14.6](https://github.com/dogecoin/dogecoin/pull/3011), except for `fr_CA`, as that has been worked on in the meantime and people keep on asking about French-Canadian. I have kept that one for now, despite that we've in the past had come to [the conclusion about this being extremely close to `fr`](https://github.com/dogecoin/dogecoin/pull/3123#issuecomment-1283088237).
- Updates all remaining translations.

The full list of removed translations:

- bg_BG
- ca@valencia
- ca_ES
- el_GR
- es_AR
- es_CL
- es_CO
- es_DO
- es_ES
- es_MX
- es_UY
- es_VE
- et_EE
- fa_IR
- fr_FR
- ro_RO
- ru_RU
- tr_TR
- vi_VN
- zh
- zh_HK

Please note that the string freeze has been generated on top of #3412 and is blocked until that PR is in, or rejected and then I will redo this work.

### CODE REVIEW

**Comment by patricklodder** [comment]
Marked as `urgent` because this is a prerequisite for translators to start. If anything needs to change we can do that separately.

File: src/qt/locale/bitcoin_en.ts [hunk 1]
```diff
@@ -2097,12 +2105,20 @@ p, li { white-space: pre-wrap; }
...
-        <location line="+13"/>
+        <location line="+9"/>
+        <location line="+14"/>
         <source>Payment request file handling</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="-14"/>
+        <source>Payment request file handling is disabled by default.Restart with -enable-bip70 if you absolutely have to use this functionality.
```
Comment (slightlyskepticalpotat) [inline src/qt/locale/bitcoin_en.ts:2115]: space here?

**Review by slightlyskepticalpotat — APPROVED** [review]
image
Assuming the latter two commits are autogenerated, the first commit looks good!

File: src/qt/locale/bitcoin_en.ts [hunk 2]
```diff
@@ -2070,19 +2070,27 @@ p, li { white-space: pre-wrap; }
...
+        <location line="+20"/>
         <location line="+14"/>
         <location line="+7"/>
         <source>URI handling</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-20"/>
+        <location line="-41"/>
+        <source>BIP70 payment requests are deprecated and disabled by default.Restart with -enable-bip70 if you absolutely have to use this functionality.
```
[the remark this answers is not in the document]
Comment (patricklodder) [reply src/qt/locale/bitcoin_en.ts:2087]: It's like that in the source. I can prepend a commit that fixes the source.

**Comment by patricklodder** [comment]
Per @slightlyskepticalpotat's comment, I have a new branch ready to fix the missing spaces from #3412. This is because base translation files are generated from sources too, with `make -C src translation_base`.

- Currently this PR has been saved in https://github.com/patricklodder/dogecoin/tree/1.14.7-translation-base-1
- The fix for the missing spaces is sitting at c045bba37, in https://github.com/patricklodder/dogecoin/tree/1.14.7-translation-base-2 - note that this is rebased on top of 4a544d65.

Full diff: https://github.com/patricklodder/dogecoin/compare/1.14.7-translation-base-1...patricklodder:dogecoin:1.14.7-translation-base-2

```
% git range-diff --short 1.14.7-translation-base-1...1.14.7-translation-base-2
-:  --------- > 1:  e59e3ab34 Disable BIP-70 payment server by default
-:  --------- > 2:  581073df4 build: Update AX_PTHREAD
-:  --------- > 3:  c045bba37 fix missing space in BIP70 warning
2:  5d20abe0f = 4:  6de4d2284 locale: remove obsolete translation files
1:  01d773612 ! 5:  0951ae369 locale: Update base translation files
3:  e59ee0c48 ! 6:  fca8fc9c9 locale: update translations
```

I can easily hang the `-2` into this PR, let me know what is desired.

**Comment by slightlyskepticalpotat** [comment]
At least to me, it would make the most sense to hang it off the same PR.

**Comment by patricklodder** [comment]
Since this is urgent, I've pushed the branch with the fixed spaces now. Can reverse if so desired...

**Comment by slightlyskepticalpotat** [comment]
I've started translating with the files from that branch. Just one thing:
is there utility to maintaining zh_TW? It's in traditional Chinese
characters as opposed to zh_CN, but less people learn it and mutual
intelligibility is high (so someone who only understands traditional
Chinese should be able to use zh_CN).

**Comment by patricklodder** [comment]
> is there utility to maintaining zh_TW?

It simply wasn't on the list last time, though I think that that was by accident - the only change to it looks like sed. This release is too urgent to do another round of depreciations - so I'm all for minimizing files to maintain, so perhaps we can arrange something for the next release.

**Comment by patricklodder** [comment]
> Just to clarify, is France going to run off the base French file, and Canadians have the option to run off the CA dialect file?

I am assuming that they speak French in France, yes.

**Comment by patricklodder** [comment]
> it's comparable to the differences seen between english US and english UK.

In general I think that it's better to have one really good translation than 2 badly maintained ones. But, since every time we talk about translations here the first thing that is brought up is `fr_CA`, I didn't want to shatter anyone's dreams. We'll see if it persists.

**Comment by slightlyskepticalpotat** [comment]
Is there any timeline for when this PR will be merged? Waiting on this
before I can PR some updated translations.

**Comment by patricklodder** [comment]
> Is there any timeline for when this PR will be merged?

Not as long as there's no approval.

You can simply open your PR including the commits from here - those preceding commits will disappear if this gets merged. Otherwise we can figure out a rebase.

**Review by chromatic — APPROVED** [review]
ACK, code changes look sensible and all tests pass on x86-64 Linux.

**Pull request merged**